test/tools/router_check/validation.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package envoy.RouterCheckToolSchema;
+
+message Validation {
```

Comment:
Can you add comments to all the fields in this file? It will help it be self documenting. Thank you!